examples/server/server.cpp

```diff
@@ -1299,6 +1312,11 @@ struct server_slot {
 
     std::function<void(int)> callback_on_release;
 
+    // Speculative decoding stats
+    int32_t n_draft_total = 0;      // Total draft tokens generated
+    int32_t n_draft_accepted = 0;   // Draft tokens actually accepted
+    float draft_accept_ratio = 0.f; // n_draft_accepted/n_draft_total
```

Comment:
This var is redundant - client can compute it trivially.

Reply:
I removed `draft_accept_ratio`